vunit/ui.py

```diff
@@ -894,6 +915,7 @@ def _main_run(self, post_run):
         """
         simulator_if = self._create_simulator_if()
         test_list = self._create_tests(simulator_if)
+        self._add_ext_pkg()

```

Comment:
I do not think this belongs here.

Reply:
I know. But I could not find how and at which point do you decide to compile `*200x` or `*-93` variants of certain files. I think that the same approach might be valid here.

Reply:
This is avoided now, since external packages are added explicitly through a builtin method.